### PR TITLE
Add openSUSE Leap 15.5 as a supported client

### LINKED ISSUE #2028: Add openSUSE Leap 15.5 as a supported client
Add openSUSE Leap 15.5 as a supported client

## Documentation Update Required

Add openSUSE Leap 15.5 as a supported client

## What does the documentation say now?

Add openSUSE Leap 15.5 as a supported client

## What change is required?

What content needs to be added?
Include description of content, and potential location of new content in the docs suite.

## Subject Matter Expert (SME)
Anyone from the documentation team / susemanager documentation team.

## Version

Which version of the software is this feature/update expected to land in (if known)?
Unconfirmed.
Which version of the documentation must this feature/update land in (if known)?
Unconfirmed.
Does the content need to be backported to previous documentation versions (if known)?
When enabled by SCC, it will be eventually backported to SUSE Manager 4.3.
Probably not any more to 4.2

## Internal SUSE Reporters

This repository is public.
If you are reporting an issue that needs to remain company confidential, please use the internal Bugzilla instance instead.

### PR DESCRIPTION
**Pull request by raulillo82**
# Description

Add openSUSE Leap 15.5 as a supported client

# Target branches

Which documentation version does this PR apply to?

- [X] Master (Default)
- [x] Manager-4.3 (but *NOT YET*, wait until you get a ping from @admd)
- [ ] Manager-4.2

# Links

Partial fix for #2028 (backports to be added)

### CODE REVIEW

File: modules/client-configuration/pages/client-upgrades-uyuni.adoc [hunk 1]
```diff
@@ -10,11 +10,11 @@ In this section, we use openSUSE Leap as an example.
 +
 ----
 spacewalk-common-channels \
-opensuse_leap15_4 \
```
Comment (juliogonzalez) [inline modules/client-configuration/pages/client-upgrades-uyuni.adoc:13]: Let's not change the default example yet. Leap 15.5 is still Alpha

File: modules/client-configuration/pages/client-upgrades-uyuni.adoc [hunk 2]
```diff
@@ -24,19 +24,19 @@ opensuse_leap15_4-uyuni-client
 // These are custom channels.
 // For more information, see xref:client-configuration:clients-opensuse.adoc[].
 
-. In the {productname} Server {webui}, navigate to menu:Software[Manage > Channels] and click the [systemitem]``Uyuni Client Tools for openSUSE Leap 15.4 (x86_64)`` channel name.
+. In the {productname} Server {webui}, navigate to menu:Software[Manage > Channels] and click the [systemitem]``Uyuni Client Tools for openSUSE Leap 15.5 (x86_64)`` channel name.
```
Comment (juliogonzalez) [inline modules/client-configuration/pages/client-upgrades-uyuni.adoc:27]: Same here and for the other changes like this one.

File: modules/client-configuration/pages/clients-opensuseleap.adoc [hunk 3]
```diff
@@ -43,6 +43,9 @@ ifeval::[{suma-content} == true]
 | {opensuse} Leap 15.4
 | openSUSE Leap 15.4 x86_64
 
+| {opensuse} Leap 15.5
```
Comment (juliogonzalez) [inline modules/client-configuration/pages/clients-opensuseleap.adoc:46]: Ok, I guess we can add it here already, even if SCC does not show it :-)

**Review by juliogonzalez — CHANGES_REQUESTED** [review]
Some adjustments needed.

File: modules/client-configuration/pages/client-upgrades-uyuni.adoc (hunk 1 above)
Comment (raulillo82) [reply modules/client-configuration/pages/client-upgrades-uyuni.adoc]: Undone

File: modules/client-configuration/pages/client-upgrades-uyuni.adoc (hunk 2 above)
Comment (raulillo82) [reply modules/client-configuration/pages/client-upgrades-uyuni.adoc]: Changed

**Review by raulillo82 — COMMENTED** [review]
I believed all needed changes are undone now.

File: modules/client-configuration/pages/clients-opensuseleap.adoc [hunk 4]
```diff
@@ -130,6 +134,30 @@ ifeval::[{uyuni-content} == true]
 
 |===
 
+[cols="1,1,1,1,1,1,1,1", options="header"]
```
Comment (juliogonzalez) [inline modules/client-configuration/pages/clients-opensuseleap.adoc:137]: Having a look at https://w3.suse.de/~rosuna/docu-PRs-uyuni/leap15.5/build/en/uyuni/client-configuration/clients-opensuseleap.html, I don't think you need a separate new table for 15.5, as the list of channels is the same we had for 15.4

**Review by juliogonzalez — CHANGES_REQUESTED** [review]

File: modules/client-configuration/pages/clients-opensuseleap.adoc (hunk 4 above)
Comment (raulillo82) [reply modules/client-configuration/pages/clients-opensuseleap.adoc:137]: True, I thought the separation was for reading purposes, but missed the SLE backport channels. Will fix that.

**Review by juliogonzalez — APPROVED** [review]

**Review by keichwa — APPROVED** [review]

**Review by jcayouette — APPROVED** [review]

**Pull request merged**